data/syndicate world jobs.txt

```diff
@@ -0,0 +1,261 @@
+
+
+mission "Corporate retreat on <planet>"
+	job
+	repeat
+	deadline
+	description "An team of <bunks> engineers needs transport to a corporate retreat on the exotic planet of <stopovers>, after which they require return transport back to <origin> by <date>. Payment is <payment>."
+	passengers 7 18
+	to offer
+		random < 20
+	source
+		government Syndicate
+	stopover
+		attributes quarg volcanic frontier north south
+		distance 3 12
+		government Republic "Free Worlds" Syndicate Neutral Quarg
+	on stopover
+		dialog "For days your ship has played host to endless discussions about load calculations, tensile strengths, and thermodynamic limits. You're grateful for some peace and quiet as the team of engineers tromps out of the hatch and heads for their exotic corporate retreat. You prepare for the return journey to <origin>."
+	on complete
+		payment 1000 90
+		dialog "The team's boss thanks you for a smooth trip and authorizes a payment of <payment>."
+
+
+mission "Corporate espionage"
+	description "The <npc> is about to enter this system. Locate it and make detailed scans of its cargo, which are of considerable interest to a team of Syndicate researchers on <destination>; proceed there for <payment> after retrieving the data."
+	job
+	repeat
+	to offer
+		random < 10
+	source
+		government Republic Syndicate Neutral
+	destination
+		government Syndicate
+		attributes research
+		distance 3 12
+	npc "scan cargo"
+		government Merchant
+		personality timid fleeing uninterested entering
+		fleet
+			names "civilian"
+			variant
+				"Scout"
+			variant
+				"Bounder"
+		dialog "You manage to make the requested scans of the <npc>'s cargo. Time to head to <planet>."
+	on complete
+		dialog `You transmit your scans and receive a hearty congratulations, as well as <payment>. "This will set them back months!" you hear someone say in the background as you close the audio link.`
+		payment 50000
+
+
+mission "Document delivery to <planet>"
+	job
+	repeat
+	description "Deliver <tons> of ordinary documents to <destination>. Payment is <payment>."
+	cargo "Confidential documents" 6 12
+	to offer
+		random < 10
+	source
+		government Syndicate
+	destination
+		distance 3 20
+		government Republic "Free Worlds" Syndicate Neutral
+		attributes factory mining "dirt belt"
+	on complete
+		payment 5000 170
+		dialog "Almost as soon as you open the cargo hatch, several silent and surprisingly well-dressed men feed the crates of documents onto a conveyor belt right next to your landing site that you hadn't noticed, where to your surprise they soon disappear into the flames of an industrial incinerator. You collect your payment of <payment>."
+
+
+mission "Waste recycling on <planet>"
+	job
+	repeat
+	description "Syndicated Systems is looking for intrepid captains willing to make a difference as part of our new Green Initiative! Deliver <tons> of recyclable <commodity> to our special facility on <destination>. Payment is <payment>."
```

Comment:
I was trying to point out that it implied stuff like the player being part of the Syndicate.
> Syndicated Systems is looking for intrepid captains willing to make a difference as part of their new "Green Initiative." Deliver `<tons>` of recyclable `<commodity>` to their special facility on `<destination>`. Payment is `<payment>`.

I think this would fit in with the other missions.